frontend/src/pages/quiz/tasks/[[...filters]].tsx

```diff
@@ -66,6 +63,18 @@ const Tasks: FC<TasksProps> = ({ applyingBenefits, beforeRetiring, filters, rece
               {t('restart-quiz')}
             </Button>
           </div>
+          <p>Answers:</p>
+          <ul className="mb-4 list-disc space-y-2 pl-7">
+            {filters?.answers?.map((answer) => (
+              <li key={answer}>{answer}</li>
+            ))}
+          </ul>
+          <p>Tags:</p>
+          <ul className="list-disc space-y-2 pl-7">
+            {filters?.tags?.map((tag) => (
```

Comment:
I was under the impression filters were different from the tags.  The tags would be 
![image](https://user-images.githubusercontent.com/48450599/235449581-87216527-e0f4-4272-871b-ee29334a65fe.png)  whereas what's here is a flat array of answerIds?  I'm going to submit a PR for the task filter today that might make this section moot though.

Reply:
Left navigation would be for extra tag filtering. I displayed the tag part of the filters. Anwsers will apply answers filter then you have tags based on the task's tags as an extra filter. I might be wrong but it's my understanding since we don't have an actual elaborated requirement.

Reply:
alright, I think we can merge this for now and finalize the design/content when it's decided.

Reply:
FYI: My understanding is that each task has **one tag** and **isTimeSensitiv boolean** for filtering.